incubator/nfs-pv-provider/values.yaml

```diff
@@ -0,0 +1,14 @@
+image:
+  repository: quay.io/kingdonb/nfs-data
+  tag: master
```

Comment:
Is this image mutable? I'd feel better about this if the image used were not. i.e. Something tagged with semver that will never be overwritten would be better.

Reply:
We can use gcr.io/google-samples/nfs-server:1.1

I don't have any need to add changes to the sample image yet, but if I do diverge with gcr.io/google-samples/nfs-server I'll make it a point to use semver.

Will update the chart and retest this.

Reply:
2865480f